src/patternfly/components/SearchInput/examples/SearchInput.md

```diff
@@ -112,6 +305,66 @@ import './SearchInput.css'
     {{/search-input-menu-list}}
   {{/search-input-menu}}
 {{/search-input}}
+
+<div class="ws-example-wrapper" style="margin-top: 200px;">
+{{#> text-input-group text-input-group--id="autocomplete" text-input-group--value="app"}}
+  {{#> text-input-group-main text-input-group-main--modifier="pf-m-icon"}}
+    {{#> text-input-group-text}}
+      {{> text-input-group-icon}}
+      {{> text-input-group-text-input text-input-group-text-input--hint-text="app" }}
```

Comment:
Since this is just autocomplete, you shouldn't need to render the hint text input. The [react search input autocomplete demo](https://www.patternfly.org/v4/components/search-input/react-demos#search-with-autocomplete) only renders it when hint text is shown, and the [core search input autocomplete](https://www.patternfly.org/v4/components/search-input/html#autocomplete) core [advanced expanded autocomplete search input and text input group examples](https://patternfly-pr-4730.surge.sh/components/search-input) don't render it either.